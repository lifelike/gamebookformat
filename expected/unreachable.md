# Gamebook #


Turn to 1 to begin.
<a name="section1">

## 1 ##
 It starts here. You can get to [6](#section6)
. 
<a name="section2">

## 2 ##
 Another fake section that is also ignored. 
<a name="section3">

## 3 ##
 You can not get here either, but because it is tagged as fake there will be no warning generated. 
<a name="section5">

## 5 ##
 You can not get here. Verification should warn you about that. 
<a name="section6">

## 6 ##
 You can get here. Congratulations! 

