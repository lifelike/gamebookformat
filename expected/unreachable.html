<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'unreachable';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 It starts here. You can get to <a class="sectionref enabledlink" data-ref="6"
href="#section6">6</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section2"
     >
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 Another fake section that is also ignored. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>
<div class="section" id="section3"
     >
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 You can not get here either, but because it is tagged as fake there will be no warning generated. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script>
<div class="section" id="section5"
     >
<div class="sectionnumber" id="para5">5</div>
<div class="sectiontext">
 You can not get here. Verification should warn you about that. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(5, document.getElementById('section5'));
}
</script>
<div class="section" id="section6"
     >
<div class="sectionnumber" id="para6">6</div>
<div class="sectiontext">
 You can get here. Congratulations! 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(6, document.getElementById('section6'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
